hack/run

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env bash
+
+set -ex
+
+ROOT_DIR=$(dirname $0)/..
+
+action="$1"
+
+if [[ "$action" == "deploy-infra" ]]; then
+  # Install dependencies and publish test images
+	source "${ROOT_DIR}"/test/e2e-common.sh && knative_setup
+elif [[ "${action}" == "teardown-infra" ]]; then
+  # Delete dependencies
+	source "${ROOT_DIR}"/test/e2e-common.sh && knative_teardown
+elif [[ "${action}" == "deploy" ]]; then
+  # Install Eventing Kafka Broker
+	source "${ROOT_DIR}"/test/e2e-common.sh && test_setup
+elif [[ "${action}" == "teardown" ]]; then
+  # Delete Eventing Kafka Broker
+	source "${ROOT_DIR}"/test/e2e-common.sh && test_teardown
+elif [[ "${action}" == "unit-test" || "${action}" == "unit-tests" ]]; then
+  # Run unit tests
+	"${ROOT_DIR}"/test/presubmit-tests.sh --unit-tests
+elif [[ "${action}" == "unit-test-data-plane" || "${action}" == "unit-tests-data-plane" ]]; then
+  # Run data plane unit tests
+	cd "${ROOT_DIR}"/data-plane && ./mvnw verify -B -U && cd -
+elif [[ "${action}" == "unit-test-control-plane" || "${action}" == "unit-tests-control-plane" ]]; then
+  # Run control-plane unit tests
+	go test -race ./...
+elif [[ "${action}" == "build-test" || "${action}" == "build-tests" ]]; then
+  # Run build tests
+	"${ROOT_DIR}"/test/presubmit-tests.sh --build-tests
+elif [[ "${action}" == "integration-test" || "${action}" == "integration-tests" ]]; then
+  # Run integration tests
+	source "${ROOT_DIR}"/test/e2e-common.sh && go_test_e2e -timeout=30m ./test/e2e_new/...
+	source "${ROOT_DIR}"/test/e2e-common.sh && go_test_e2e -timeout=30m ./test/e2e/...
+	source "${ROOT_DIR}"/test/e2e-common.sh && go_test_e2e -tags=deletecm ./test/e2e/...
```

Comment:
Same comment i did in the makefile, let's have the actual "test commands" in the proper script. 

The reason is that the knative developers expect those commands in `e2e-tests.sh`, not here.

Also, if I'm not mistake, you're creating a circular ref between scripts. Although this technically doesn't break, it definitely breaks my mind :laughing:. Let's keep this script just an "interface" to run scripts, but then the actual logic where it belongs

Reply:
sgtm, done!